README.md

```diff
@@ -14,7 +14,7 @@ PFRL is a deep reinforcement learning library that implements various state-of-t
 
 ## Installation
 
-PFRL is tested with 3.6+. For other requirements, see [requirements.txt](requirements.txt).
```

Comment:
'+' should be removed since our CI use 3.7.7 only right now.

Reply:
Also, I think it is better to write `Python 3.7.7`, not just `3.7.7`. It is a separate issue though.

Reply:
Done.